lib/events.js

```diff
@@ -207,18 +217,22 @@ exports.deleteEvent = async (req, res) => {
 };
 
 exports.setApprovalStatus = async (req, res) => {
-    if (!req.permissions.change_status[req.body.status]) {
+    if (!req.permissions.change_status[req.body.status.replace(' ', '_')]) {
         return errors.makeForbiddenError(res, 'You are not allowed to change status.');
     }
 
     const oldStatus = req.event.status;
 
     await sequelize.transaction(async (t) => {
-        await req.event.update({ status: req.body.status }, { transaction: t });
+        const event = req.event;
+        event.organizers = await Promise.all(event.organizers.map((organizer) =>
+            core.fetchUser(organizer, req.headers['x-auth-token'])));
```

Comment:
@serge1peshcoff in order to always have the proper notification_email I added this. what do you think about it?

Reply:
I'd say so, but keep in mind this may fail if the user doesn't have the permissions to view a user

Reply:
That permission is available for the General members circle so that should not be a problem, but good to keep in the back of our minds